apps/web/src/screens/HomePage/sections/SIGs/index.tsx

```diff
@@ -131,29 +58,52 @@ const Line: AnyStyledComponent = styled.hr`
   margin: 100px 0;
 `;
 
-const SIGs: React.FC = (): JSX.Element => {
+const SIGs: React.FC<{}> = (): JSX.Element => {
+  const result: GetSigsQueryHookResult = useGetSigsQuery();
+  let SIGsData: ISIG[] = [];
+  var isLoading;
+  if (result.data && result.data.sigs) {
+    SIGsData = result.data.sigs.filter(sig => sig.display) as ISIG [];
+    SIGsData.sort(
+      (a: ISIG, b: ISIG) =>
+      {
+        let filterResult;
+        if(a.name.toLowerCase() > b.name.toLowerCase()) filterResult = 1;
+        else if(a.name.toLowerCase() < b.name.toLowerCase()) filterResult = -1;
+        else filterResult = 0;
+        return filterResult;
+      });
+    isLoading = false;
+  } else {
+    isLoading = true;
+  }
```

Comment:
This code should be in a useEffect https://reactjs.org/docs/hooks-effect.html where the code would rely on result being modified, and SIGsData should be state instead of a variable

Reply:
Loading should also be a variable

Reply:
if he's not refetching why should he use an effect hook?